dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/DatadogCore.kt

```diff
@@ -268,6 +269,10 @@ internal class DatadogCore(
         return coreFeature.createScheduledExecutorService(executorContext)
     }
 
+    override fun setAnonymousId(anonymousId: UUID?) {
```

Comment:
> [!TIP]
> Do we need nullability support here? if `anonymousId` == null then `anonymousId .toString()` will return "null" so if someone will set null it actually will similar to set "null". Not sure if it's an issue, but it's seems like `null` should drop the value instead

Reply:
Yes - when developer using the SDK decided disabling this feature, we would clear the id.